R/dummy_functions.R

```diff
@@ -13,7 +13,7 @@
 #'   shinyApp(app$ui, app$server)
 #' }
 #' @export
-example_module <- function(label = "example teal module", datanames = "all", transformers = list()) {
+example_module <- function(label = "example teal module", datanames = "all", transforms = list()) {
```

Comment:
I keep having a personal fight looking at whether we should use `transformers` or `transforms`.
`Transforms` sound like a noun and could be used to name the action of transforming, e.g.:

> "This function transforms the input image by applying a series of filters and scaling operations."

However, when we think about a verb, I reckon `transformers` is the more appropriate name

> "The data pipeline includes several transformers that clean and normalize the data before it is fed into the machine learning model.

I would go with `transformers` in the documentation and naming as this is the name for objects/verbs. Transforms in my opinion is the name for the noun/action